test/fail_compilation/fail_scope.d

```diff
@@ -25,6 +21,10 @@ fail_compilation/fail_scope.d(137): Error: returning `foo16226(i)` escapes a ref
 //fail_compilation/fail_scope.d(40): Error: scope variable p may not be returned
 */
 
+
+
+
+
```

Comment:
I add them sometimes so I can add test cases in TEST_OUTPUT without having to renumber them. I learned that when programming in BASIC :-)